rocky/rocky/templates/organizations/organization_crisis_room.html

```diff
@@ -26,7 +26,8 @@ <h1>{% translate "Crisis room" %}: {{ organization.name }}</h1>
                     {% translate "An overview of the top 10 most severe findings OpenKAT found. Check the detail section for additional severity information." %}
                 </p>
                 <h2>{% translate "Top 10 most severe Findings" %}</h2>
-                {{ object_type_filters }}
+                {% include "partials/show_active_filters.html" %}
+
```

Comment:
There is no filter on this page. `object_type_filter` didn't do anything because it was not set before, and the partial also won't do anything because `active_filters` isn't defined. Better to just remove this from the template.